components/Error/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment } from 'react'
+import { graphql, compose } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import withT from '../../lib/withT'
+import { Router } from '../../lib/routes'
+
+import Loader from '../Loader'
+import Me from '../Auth/Me'
+import Meta from '../Frame/Meta'
+
+import ErrorFrame from './Frame'
+
+import {
+  Interaction
+} from '@project-r/styleguide'
+
+const getRedirect = gql`
+query getRedirect($path: String!) {
+  redirection(path: $path) {
+    target
+    status
+  }
+}
+`
+
+const ErrorComponent = ({statusCode, t, loading, children}) => (
```

Comment:
Maybe name this something like StatusError? ErrorComponent could be all kinds of things so a more specific name could be helpful moving forward.